ale/drivers/kaguya_drivers.py

```diff
@@ -1,14 +1,18 @@
-import spiceypy as spice
 import os
-import pvl
-import ale
-
+from glob import glob
```

Comment:
:+1: Agreed on moving that globing out. Although, would this `_metakernel_dir` replace the config.py? Therefore moving instrument config to the instrument specific module? I imagine when we do kernel selection properly, we could get any meta data from the driver for queries? Since it would all be label data.